config/fieldparams/mainnet.go

```diff
@@ -39,4 +39,13 @@ const (
 	MaxDepositRequestsPerPayload          = 8192          // Maximum number of deposit requests in an execution payload.
 	MaxWithdrawalRequestsPerPayload       = 16            // Maximum number of execution layer withdrawal requests in an execution payload.
 	MaxConsolidationRequestsPerPayload    = 1             // Maximum number of consolidation requests in an execution payload.
+	MaxProposerSlashings                  = 16            // Maximum number of proposer slashings in a block.
+	MaxAttesterSlashings                  = 2             // Maximum number of attester slashings in a block.
+	MaxAttesterSlashingsElectra           = 1             // Maximum number of attester slashings in a block.
+	MaxAttestations                       = 128           // Maximum number of attestations in a block.
+	MaxAttestationsElectra                = 8             // Maximum number of attestations in a block.
+	MaxDeposits                           = 16            // Maximum number of deposits in a block.
+	MaxVoluntaryExits                     = 16            // Maximum number of voluntary exits in a block.
+	MaxValidatorsPerWithdrawalsSweep      = 16            // Maximum number of validators that can be processed in a single withdrawals sweep.
+	MaxBlsToExecutionChanges              = 16            // Maximum number of bls to execution changes in a block.
```

Comment:
You have to add this to minimal too. I see that it is not added to `minimal_config.go`, which is a mistake. Can you add it while you're at it?

Reply:
> You have to add this to minimal too. I see that it is not added to `minimal_config.go`, which is a mistake. Can you add it while you're at it?

I added it to `minimal.go` and `mainnet.go`. Okay I'll add it to the one you said too

Reply:
> You have to add this to minimal too. I see that it is not added to `minimal_config.go`, which is a mistake. Can you add it while you're at it?

Aren't the values already mentioned in `minimal_config.go`?
https://github.com/prysmaticlabs/prysm/blob/af098e737ef3fa5ae2eb5b069607d1f016ff64c9/config/params/minimal_config.go#L65-#L73

Except the constant `MaxBlsToExecutionChanges`